.github/workflows/build-and-test-ngas.yml

```diff
@@ -31,6 +31,10 @@ jobs:
           test_number: 4
         - python: 3.9
           test_number: 5
+        - python: 3.9
+          db: mysql
+          sql_mode: no_prepared_statement
```

Comment:
Suggestion: name this parameter `use_prepared_statements` (using plural here, but see my other comment), rename the env variable `SQL_MODE` to `USE_PREPARED_STATEMENTS` in the `Run NGAS tests` step, finally calculate `use_prepared_statements_attr='UsePreparedStatements="'${USE_PREPARED_STATEMENTS:-true}'"'` in the `run_tests.sh` script and always append the attribute to the `NGAS_TESTDB` definition. That makes it a bit less mysql-specific, and keeps the naming consistent across all the layers.

BTW, have you run locally the test suite with this new option set against a Sybase DB?

Reply:
I've implemented all your suggestions. The reason for having `sql_mode=no_prepared_statements` was to display something more meaningful than `false` in the test execution summary in github, but I agree that consistency across the code is probably more important.
I have not run the tests against Sybase but I'm running manual tests on a pre-production system and this is how I found the issue with passing empty args to `cursor.execute()`